src/ArmTemplates/Common/API/Clients/Tags/TagClient.cs

```diff
@@ -58,20 +58,12 @@ public async Task<List<TagTemplateResource>> GetAllTagsLinkedToApiAsync(string a
 
         public async Task<List<TagTemplateResource>> GetAllTagsLinkedToProductAsync(string productName, ExtractorParameters extractorParameters)
         {
-            var apisLinkedToProduct = await this.apisClient.GetAllLinkedToProductAsync(productName, extractorParameters);
-
-            var tagTemplateResources = new List<TagTemplateResource>();
-            foreach (var productApi in apisLinkedToProduct)
-            {
-                var apiTags = await this.GetAllTagsLinkedToApiAsync(productApi.Name, extractorParameters);
+            (string azToken, string azSubId) = await this.Auth.GetAccessToken();
```

Comment:
just fyi - you can simplify the expression
```suggestion
            var (azToken, azSubId) = await this.Auth.GetAccessToken();
```